content/blog/2018-03-13-weathercan.md

```diff
@@ -0,0 +1,500 @@
+
+First, site A:
+
+``` r
+siteA <- stations_search(coords = c(49.688211, -97.116088), 
+                         interval = "day", dist = 25) %>%
+  select(-prov, -climate_id, -WMO_id, -TC_id) %>%
+  filter(end >= 2018)
+siteA
+```
+
+    ## # A tibble: 3 x 9
+    ##   station_name  station_id   lat   lon  elev interval start   end distance
+    ##   <chr>         <fct>      <dbl> <dbl> <dbl> <chr>    <int> <int>    <dbl>
+    ## 1 WINNIPEG CHA… 43185       49.8 -97.3   238 day       2004  2018     21.1
+    ## 2 WINNIPEG THE… 28051       49.9 -97.1   230 day       1999  2018     22.5
+    ## 3 KLEEFELD (MA… 50897       49.5 -96.9   246 day       2014  2018     24.4
+
+Now, site B:
+
+``` r
+siteB <- 
+  stations_search(coords = c(49.267330, -97.327142), 
+                  interval = "day", dist = 35) %>%
+  select(-prov, -climate_id, -WMO_id, -TC_id) %>%
+  filter(end >= 2018)
+siteB
+```
+
+    ## # A tibble: 2 x 9
+    ##   station_name station_id   lat   lon  elev interval start   end distance
+    ##   <chr>        <fct>      <dbl> <dbl> <dbl> <chr>    <int> <int>    <dbl>
+    ## 1 EMERSON AUTO 48068       49.0 -97.2   242 day       2009  2018     30.4
+    ## 2 GRETNA (AUT) 3605        49.0 -97.6   253 day       1885  2018     31.4
+
+We have a selection of stations for each site that are all about the same distance away. Before we chose any we should make sure they have the data we're interested in.
+
+Above, we already downloaded all the data for Manitoba in this date range (`mb_weather`), so we can `select()` the types of data we want and then `filter()` to the specific `station_id`s we're interested in. We can then use the `naniar` package to easily visualize any missing data from these stations:
+
+``` r
+# Limit ourselves to temperature and precipitation
+mb_weather <- select(mb_weather_all,
+                     station_id, lat, lon,
+                     date, min_temp, mean_temp, max_temp,
+                     total_precip, total_snow)
+```
+
+``` r
+# Stations near site A (< 25 km)
+mb_weather %>%
+  filter(station_id %in% siteA$station_id) %>%
+  gg_miss_var(show_pct = TRUE, facet = station_id) + 
+  labs(title = "Site A: Stations < 25 km")
+```
+
+<center>
+<img src = "/img/blog-images/2018-03-13-weathercan/naniarA-1.png" style = "width: 80%">
+</center>
+<p>
+``` r
+# Stations near site B (< 35 km)
+mb_weather %>%
+  filter(station_id %in% siteB$station_id) %>%
+  gg_miss_var(show_pct = TRUE, facet = station_id) + 
+  labs(title = "Site B: Stations < 35 km")
+```
+
+<center>
+<img src = "/img/blog-images/2018-03-13-weathercan/naniarB-1.png" style = "width: 80%">
+</center>
+<p>
+No station has much snow data, unles we're willing to lose all temperature data. For site A, it definitely looks like station id 28051 has the most complete data (temperature and precipitation, at least). Site B we can use either station.
```

Comment:
unless

Reply:
this sentence isn't clear: "No station has much snow data, unles we’re willing to lose all temperature data." I think it might help to add a line or two